base/xx-go

```diff
@@ -9,23 +9,45 @@ done
 export GOOS=${TARGETOS}
 export GOARCH=${TARGETARCH}
 
-if [ "$TARGETARCH" = "arm" ]; then
-  if [ -n "$TARGETVARIANT" ]; then
-    case "$TARGETVARIANT" in
-      "v5")
-        export GOARM="5"
-        ;;
-      "v6")
-        export GOARM="6"
-        ;;
-      *)
-        export GOARM="7"
-        ;;
-    esac
-  else
-    export GOARM="7"
-  fi
-fi
+case "$TARGETARCH" in
+  "amd64")
+    if [ -n "$TARGETVARIANT" ]; then
```

Comment:
Maybe if `GOAMD64` is already set to a valid value then this should not overwrite (same for `GOARM`)